appendix.md

```diff
@@ -0,0 +1,26 @@
+# Browser Columns
+
+All browser columns are available for both
+[Cards and Notes mode](browsing.md#table-modes) but sometimes with slightly
+different names and data. The following table lists the behaviours for both modes.
+The list of cards or notes can be sorted by every column except for the
+Question and Answer columns.
+
+
+| Column | Cards mode | Notes mode |
+|-|-|-|
+| Answer | Shows the back side of the card in one line with the question stripped. You can also choose a [custom format](templates/styling.md#browser-appearance) in the card type editor instead. | Shows the same as in Cards mode, only for the first card of the note. |
+| Card(s) | Shows the name of the card's template. | Shows the number of cards the note has. |
+| Card Modified | Shows the last time changes were made to the card (e.g., when you reviewed the card and the review history and interval were updated). | Shows the last time changes were made to one of the note's cards. |
+| Created | Shows the same as in Notes mode for the card's note. | Shows when the note was created. |
+| Deck | Shows the name of the deck the card is in. | Shows the number of different decks the note's cards are in, or the deck name if it's only one. |
+| Due | Shows the due date for cards in reviews or (re)learning and the position in the new card queue for new cards. The line is wrapped in brackets if the card is suspended or buried. Sorting is done by type and only then by date or position. | Shows the due date for the note's next due review or (re)learning card that is not suspended, buried or in a filtered deck. |
+| (Avg.) Ease | Shows the card's ease unless it's a new card. | Shows the average ease for the note's cards that are not new. |
+| (Avg.) Interval | Shows the card's interval if the card is in reviews or relearning. | Shows the average interval for the note's cards that are in reviews or relearning. |
+| Lapses | Shows how often the card lapsed. | Shows how often the note's cards have lapsed altogether. |

```

Comment:
"...the card was answered Again" might be clearer to the layperson

"the total lapses for all cards of the note"?